drizzlepac/haputils/hapcut_utils.py

```diff
@@ -492,32 +602,50 @@ def mvm_combine(cutout_files, img_combiner=None, output_dir=".", log_level=logut
         The desired level of verbosity in the log statements displayed on the screen and written to the
         .log file. Default value is 20, or 'info'.
 
-    Note: The combiner only works with cutouts which have been generated as multiple output files,
-    rather than one large file with many cutout extensions.
-
     """
 
-    # Make sure the cutout_files are really a list of MULTIPLE filenames
-    if type(cutout_files) == str or type(cutout_files) == list and len(cutout_files) < 2:
-        log.error("The 'mvm_combine' function requires a list of MULTIPLE cutout filenames where" \
-                  " the files were generated by 'make_the_cut' with the parameter single_output=False.")
+    # set logging level to user-specified level
+    log.setLevel(log_level)
 
-    # Report the cutout files to be used for the combination process
-    log.info("Input cutout files:")
-    for cf in cutout_files:
-        log.info("{}".format(cf))
-
-    # Cutout filenames: hst_cutout_skycell-p0081-ra84d920799-decs69d851699[_uvis_f275w].fits
-    # Construct the combined filename based on the first file in the list as only RA and Dec are used
-    output_prefix = "hst_combined_skycells-"
-    filename = fits.getheader(cutout_files[0], ext=0)['FILENAME']
-    tokens = filename.split("_")[2].split("-")
-    skycell = tokens[1][1:5]
-    ra = tokens[2]
-    dec = tokens[3]
-    output_filename = os.path.join(output_dir, output_prefix + ra + "-" + dec + ".fits")
-
-    # Combine the SCI and then the WHT extensions in the specified files
-    log.info("Combining the SCI and then the WHT extensions of the input cutout files.")
-    combined_cutout = astrocut.CutoutsCombiner(cutout_files, img_combiner=img_combiner).combine(output_file=output_filename)
-    log.info("The combined output filename is {}.".format(output_filename))
+    # Output prefix
+    OUTPUT_PREFIX = "hst_combined_skycells-"
+
+    for key, file_list in input_dict.items():
+
+        # If there are multiple files to combine, then do it
+        if len(file_list) > 1:
+
+            # Construct the combined filename based on the first file in the list
+            # Example: hst_combined_skycells-ra84d9207-decs69d8516_uvis_f275w.fits
+            filename = fits.getheader(file_list[0], ext=0)['FILENAME']
```

Comment:
Please confirm the format of the input filenames.  I would expect that the input files to be MVM product filenames of the format `hst_skycell-p0081x14y14_acs_wfc_f658n-all_drc.fits`.  Perhaps I misunderstand how this code was supposed to be called or when it was supposed to be used.